### PR TITLE
Create demo snippets for the Duration type

### PR DESCRIPTION
**Pull request by carlosschults**

### CODE REVIEW

**Comment by jskeet** [comment]
Just to let you know, I've suddenly become massively busy. Will try to review these over the weekend, but it may well not be before then. Thanks for them though - please don't take my lack of review as a lack of appreciation :)

**Comment by carlosschults** [comment]
@jskeet No problem ;)

File: src/NodaTime.Demo/DurationDemo.cs [hunk 1]
```diff
@@ -0,0 +1,89 @@
+﻿// Copyright 2010 The Noda Time Authors. All rights reserved.
```
Comment (jskeet) [inline src/NodaTime.Demo/DurationDemo.cs]: 2017

File: src/NodaTime.Demo/DurationDemo.cs [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+        [Test]
+        public void ConstructionFromTimeSpan()
+        {
+            Duration duration = Snippet.For(Duration.FromTimeSpan(TimeSpan.FromHours(3)));
+            Assert.AreEqual(3, duration.Hours);
+            Assert.AreEqual("0:03:00:00", duration.ToString());
+        }
+
+        [Test]
```
Comment (jskeet) [inline src/NodaTime.Demo/DurationDemo.cs]: I'd get rid of this one; Duration is a value type so there's no user-defined parameterless constructor anyway.

**Comment by carlosschults** [comment]
@jskeet Corrections made as asked. Thanks :)

**Pull request merged**